README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
```

Comment:
@Logan1x Move this line below line 2. Below the `# Open Source Help Community` line.